.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,7 +9,7 @@ Resolves #
 
 **Checklist:**
 
-- [ ] **CommitLint Bot**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/master/development-flow.html#commit-structure).
+- [ ] **Commitlint / lint (pull_request) Github Action**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/master/development-flow.html#commit-structure).
```

Comment:
What about this title so it might be more meaningful or understandable as something they should look at?
```suggestion
- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/master/development-flow.html#commit-structure).
```